releases/release-1.20/release-notes-draft.md

```diff
@@ -1,178 +1,258 @@
 - Introduces a metric source for HPAs which allows scaling based on container resource usage. ([#90691](https://github.com/kubernetes/kubernetes/pull/90691), [@arjunrn](https://github.com/arjunrn)) [SIG API Machinery, Apps, Autoscaling and CLI]
-- Kube-apiserver now deletes expired kube-apiserver Lease objects: ([#95895](https://github.com/kubernetes/kubernetes/pull/95895), [@roycaihw](https://github.com/roycaihw)) [SIG API Machinery, Apps, Auth and Testing]
+- Kube-apiserver now deletes expired kube-apiserver Lease objects:
   - The feature is under feature gate `APIServerIdentity`.
-  - A flag is added to kube-apiserver: `identity-lease-garbage-collection-check-period-seconds` 
+  - A flag is added to kube-apiserver: `identity-lease-garbage-collection-check-period-seconds` ([#95895](https://github.com/kubernetes/kubernetes/pull/95895), [@roycaihw](https://github.com/roycaihw)) [SIG API Machinery, Apps, Auth and Testing]
 - Kube-controller-manager: volume plugins can be restricted from contacting local and loopback addresses by setting `--volume-host-allow-local-loopback=false`, or from contacting specific CIDR ranges by setting `--volume-host-cidr-denylist` (for example, `--volume-host-cidr-denylist=127.0.0.1/28,feed::/16`) ([#91785](https://github.com/kubernetes/kubernetes/pull/91785), [@mattcary](https://github.com/mattcary)) [SIG API Machinery, Apps, Auth, CLI, Network, Node, Storage and Testing]
 - Migrate scheduler, controller-manager and cloud-controller-manager to use LeaseLock ([#94603](https://github.com/kubernetes/kubernetes/pull/94603), [@wojtek-t](https://github.com/wojtek-t)) [SIG API Machinery, Apps, Cloud Provider and Scheduling]
 - Modify DNS-1123 error messages to indicate that RFC 1123 is not followed exactly ([#94182](https://github.com/kubernetes/kubernetes/pull/94182), [@mattfenwick](https://github.com/mattfenwick)) [SIG API Machinery, Apps, Auth, Network and Node]
 - Move configurable fsgroup change policy for pods to beta ([#96376](https://github.com/kubernetes/kubernetes/pull/96376), [@gnufied](https://github.com/gnufied)) [SIG Apps and Storage]
-- New flag is introduced to kubelet, i.e. `--topology-manager-scope=container|pod`.
+- New flag is introduced, i.e. --topology-manager-scope=container|pod. 
   The default value is the "container" scope. ([#92967](https://github.com/kubernetes/kubernetes/pull/92967), [@cezaryzukowski](https://github.com/cezaryzukowski)) [SIG Instrumentation, Node and Testing]
 - New parameter `defaultingType` for `PodTopologySpread` plugin allows to use k8s defined or user provided default constraints ([#95048](https://github.com/kubernetes/kubernetes/pull/95048), [@alculquicondor](https://github.com/alculquicondor)) [SIG Scheduling]
 - NodeAffinity plugin can be configured with AddedAffinity. ([#96202](https://github.com/kubernetes/kubernetes/pull/96202), [@alculquicondor](https://github.com/alculquicondor)) [SIG Node, Scheduling and Testing]
 - Promote RuntimeClass feature to GA.
```

Comment:
For this one, do we need to add some action items for users to start using the v1 API?

Reply:
or the fact it is in the API change section is sufficient?

Reply:
huh, I think we might need to -- what happens if users are still using pre-v1 API? is it going to break or just deprecation warning?

Reply:
no deprecation warning if they are using beta. Alpha has deprecation warning for a few releases now. Good point though, maybe when we will place a deprecation warning it will be time to ask to take an action...

Reply:
okay yeah let's add this to deprecation warning and action required. I think going to v1 is quite a big deal that current users should start planning their upgrade strategy.